### PR TITLE
Bugfix - A rule defined by not available template must be in NOT_INIT…

### PR DESCRIPTION
**Pull request by danchom**
…IALIZED state

(before the fix it goes into IDLE state).

Signed-off-by: Yordan Mihaylov <j.mihailov@prosyst.com>

### CODE REVIEW

**Comment by danchom** [comment]
Could you check this PR. It contains important fixes for Automation module.

File: bundles/automation/org.eclipse.smarthome.automation.providers/src/main/java/org/eclipse/smarthome/automation/internal/core/provider/TemplateResourceBundleProvider.java [hunk 1]
```diff
@@ -156,7 +149,7 @@ public void close() {
         if (tracker != null) {
             tracker.close();
             tracker = null;
-            moduleTypeRegistry = null;
+            // moduleTypeRegistry = null;
```
Comment (maggu2810) [inline bundles/automation/org.eclipse.smarthome.automation.providers/src/main/java/org/eclipse/smarthome/automation/internal/core/provider/TemplateResourceBundleProvider.java]: Remove this line completely as there is no such variable anymore.

**Comment by maggu2810** [comment]
Just one minor comment. Will merge this PR if fixed.

**Comment by danchom** [comment]
Done

**Pull request merged**